_config.yml

```
@@ -1,96 +1,25 @@
-# Site settings
-title: "The Union of UK Unicyclists"
-email: webmaster@unicycle.org.uk
-description: > # this means to ignore newlines until "baseurl:"
-  The website of the Union of UK Unicyclists
-baseurl: "" # the subpath of your site, e.g. /blog/
-url: "http://www.unicycle.org.uk" # the base hostname & protocol for your site
-twitter_username: UUUnicyclists
-github_username:  unionofukunicyclists
-facebook_page: theunionofukunicyclists
-google_maps_api_key: "AIzaSyBoPKA9DXHMbdvaqMPWxXGX1rUbAJN5JsI"
+# site settings
+title: The Union of UK Unicyclists
+shorttitle: UUU
+subtitle: The Union of UK Unicyclists
+email: committee@unicycle.org.uk
+description: The Union of UK Unicyclists promotes the sport and art of unicycling in the UK. 
+baseurl: /unionofukunicyclists.github.io # the subpath of your site, e.g. /blog
```

This is wrong. Please this back to how it was before. When running locally this puts everything at `localhost:4000/unionofukunicyclists.github.io`.
```suggestion
baseurl: /unionofukunicyclists.github.io # the subpath of your site, e.g. /blog
baseurl: ""
```

---

@mchonofsky I looks you still haven't fixed this. We can't merge this unless this is fixed.